### ISSUE TITLE
Instrument release rate bug

### ISSUE THREAD

**Issue by papiezak** [rerrahkr/BambooTracker#115]
It's rather minor, but anyway, steps to replicate:
1. Create an instrument, preferrably set release rates to 15 on all operators
2. Play a note, don't release the key
3. Right click anywhere
4. Release a key... and a note will still play infnitely, even while improvizing. Muting channel/track won't have any effect, only way is to kill sound with F12.

**Comment by rerrahkr** [comment]
Thanks for your report. `JamManager` monitors key press/release pairs and control jamming note on/off and its channel assignment. Context menu is opened, however, any keyboard event is ignored, then it cannot stop the sound except to press F12. I will change the channel assignment to use an existing channel when the "pressed key" is pressed.

**Comment by rerrahkr** [comment]
I fixed above at db51d5bcfe4b6534b6c90baf0ad342b3624c3e1d.

**Comment by papiezak** [comment]
That's strange, it just the same after I pulled the fresh dev build

**Comment by rerrahkr** [comment]
I only fix to release key when pressing and releasing the same key again. Context menus still block key event because it is modal. As this seems to be difficult to resolve, it may be left as it is...

**Comment by papiezak** [comment]
>I only fix to release key when pressing and releasing the same key again.

Yeah that one works fine. About the context menus hanging the key, IDK. I will leave this issue open for a while.
